app/angular/src/builders/utils/run-compodoc.ts

```diff
@@ -1,18 +1,26 @@
 import { BuilderContext } from '@angular-devkit/architect';
 import { spawn } from 'child_process';
 import { Observable } from 'rxjs';
+import * as path from 'path';
 
 const hasTsConfigArg = (args: string[]) => args.indexOf('-p') !== -1;
 
+// path.relative is necessary to workaround a compodoc issue with
+// absolute paths on windows machines
+const toRelativePath = (pathToTsConfig: string) => {
+  return path.isAbsolute(pathToTsConfig) ? path.relative('.', pathToTsConfig) : pathToTsConfig;
+};
+
 export const runCompodoc = (
   { compodocArgs, tsconfig }: { compodocArgs: string[]; tsconfig: string },
   context: BuilderContext
 ): Observable<void> => {
   return new Observable<void>((observer) => {
+    const tsConfigPath = toRelativePath('./tsconfig.json');
```

Comment:
@shilman  🤦‍♂️ When you look at code for too long, right?

```suggestion
    const tsConfigPath = toRelativePath(tsconfig);
```